modules/assisted-installer-how-to-use-the-assisted-installer.adoc

```diff
@@ -0,0 +1,44 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id="how-to-use-the-assisted-installer_{context}"]
+= How to use the Assisted Installer
+
+The {product-title} link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[{ai-full}] is a user-friendly installation solution offered on the link:http://console.redhat.com[Red Hat Hybrid Cloud Console]. The {ai-full} supports the various deployment platforms with a focus on bare metal.
+
+The {ai-full} provides installation functionality as a service. This software-as-a-service (SaaS)  approach has the following advantages:
+
+* *Web user interface:* The web user interface performs cluster installation without the user having to create the installation configuration files manually.
+* *No bootstrap node:* A bootstrap node is not required when installing with the {ai-full}. The bootstrapping process executes on a node within the cluster.
+* *Hosting:* The {ai-full} hosts:
+  - Ignition files
+  - The installation configuration
+  - A discovery ISO
+  - The installer
+* *Simplified deployment:* Deployment does not require in-depth knowledge of {product-title}. The {ai-full} provides reasonable defaults. Additionally, the {ai-full} performs rich pre-flight validation before installation to catch configuration-related installation errors before they occur.
```

Comment:
```suggestion
* *Simplified deployment:* Deployment does not require in-depth knowledge of {product-title}. The {ai-full} provides reasonable defaults. Additionally, the {ai-full} performs rich pre-flight validations before installation to catch configuration-related installation errors before they occur.
```

Reply:
Fixed, but I consolidated this with streamlined workflow bullet.